Dockerfile

```diff
@@ -24,22 +24,22 @@ RUN apt-get update && \
             /usr/share/doc/* /usr/share/groff/* /usr/share/info/* /usr/share/linda/* \
             /usr/share/lintian/* /usr/share/locale/* /usr/share/man/*
 
-# Install PHP with some extensions
+# Install PHP 8.2 with some extensions
```

Comment:
Maybe we can upgrade to php 8.3 directly or do we plan to make 2 version of the lib ?
8.2 will be end of active support at 31/12/2024 and receive only security fixed until end of 2025

Reply:
The philosophy of the PHP client always was "keep compatibility with the most"; so, I think it is before upgrading to 8.2 now and 8.3 in January